.github/workflows/gradle.yml

```diff
@@ -251,3 +251,18 @@ jobs:
         with:
           name: Maven
           path: ~/releases
+
+  dispatch:
+    name: dispatch
+    needs: [combine]
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: peter-evans/repository-dispatch@v3
```

Comment:
Any reason to use repo dispatch rather than workflow dispatch?

Reply:
Because that's what robotpy does?

https://github.com/robotpy/mostrobotpy/blob/868f50a724399cfdb3cc45b0067bbb20e9e7e401/.github/workflows/dist.yml#L319-L330

Reply:
Oh when it was first added to robotpy, Github hadn't invented workflow_dispatch yet.

Reply:
Do you have a recommendation for an action to trigger the workflow_dispatch? During a quick look through the actions marketplace, I didn't see any that seemed well maintained.